BasePolicies/CONTRIBUTING.md

```diff
@@ -19,6 +19,7 @@ and the Node.js project.
 
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Committer** is a subset of contributors who have been given write access to the repository.
+* A **Collaborator** is a committer, but may use more time tag and close issues, rather than commit code.
```

Comment:
I think something went wrong with your the editing- the sentence didn't come out right. Maybe I'm just loosing my mind :/